sharpd/sharp_vty.c

```diff
@@ -242,11 +274,30 @@ DEFPY (install_routes,
 		sg.r.nhop_group.nexthop = &sg.r.nhop;
 	}
 
+	/* Use single backup nexthop if specified */
+	if (backup) {
+		/* Set flag and index in primary nexthop */
+		SET_FLAG(sg.r.nhop.flags, NEXTHOP_FLAG_HAS_BACKUP);
+		sg.r.nhop.backup_idx = 0;
+
+		if (backup_nexthop4.s_addr != INADDR_ANY) {
```

Comment:
nit: `if (backup_nexthop4.s_addr)`?

Reply:
Again, at least to my eyes, this is more explicit? How strongly do you feel?